sources/packages/api/src-sql/Restrictions/Truncate-restriction.sql

```diff
@@ -0,0 +1,18 @@
+-- * This script is used by 'up' of CleanRestrictionsTable1652466821171 and 'down' of InsertRestrictions1652480204380 files.
+-- Clean restrictions table.
+-- Clean the tables that has relation with restrictions table.
+-- Clean federal_restrictions.
+DELETE FROM
+  sims.federal_restrictions;
+
+-- Clean institution_restrictions.
+DELETE FROM
+  sims.institution_restrictions;
+
+-- Clean student_restrictions.
+DELETE FROM
+  sims.student_restrictions;
+
+--Clean restrictions.
+DELETE FROM
```

Comment:
I believe that we talked about using TRUNCATE instead of DELETE to have the primary keys reset. Does something change?

Reply:
I tried Truncate. it was throwing reference errors, 
![image](https://user-images.githubusercontent.com/77353155/168680563-ca417d5c-b618-4e07-b7cf-40ea94e3929b.png)

Reply:
the reason why I don't have to use TRUNCATE CASCADE` is when I checked the create institution_restiction I saw it had also a relationship with **institution** and **notes**, i suspected that it will affect those. But i can give it a try
![image](https://user-images.githubusercontent.com/77353155/168683407-9ed88ad0-80cd-44ee-a2d0-01baaa8f4cc7.png)